### DISCUSSION TITLE
Product videos in PDP carousel

### DISCUSSION THREAD

**Discussion by matt-turnbull-5874** [bigcommerce/catalyst#646] [Feature Requests]
Currently the GraphQL API for products only allows you to get the product images, even though you have the ability to add both images and videos in the BigCommerce admin interface.

Would you be able to expand the GraphQL product schema to also include product videos?

**Comment by jamie-mckenzie** [comment]
Hello Matt,  

Thank you for bringing this to my attention!   You are correct that the graph product schema does not currently support video URLs assigned to the product in the control panel.  This is a gap that we plan to address in the very near term.   

Thank you for your participation with Catalyst and your input.
